modules/twinklexfd.js

```diff
@@ -714,6 +714,28 @@ Twinkle.xfd.setWatchPref = function twinklexfdsetWatchPref(pageobj, pref) {
 };
 
 Twinkle.xfd.callbacks = {
+	// Requires having the tag text (params.tagText) set ahead of time
+	autoEditRequest: function(pageobj, params) {
+		// Update the form
+		pageobj.getStatusElement().warn('Page protected, requesting edit');
+
+		var venue = utils.toTLACase(params.venue);
+
+		var talkTitle = new mw.Title(pageobj.getPageName()).getTalkPage().toText();
+		var editRequest = '{{Edit protected|' + talkTitle + '|answered=no}}\n' +
+			pageobj.getPageName() + ' has been listed at ' + venue + ' ([[' + params.discussionpage + '|nomination]]), but is protected so could not be tagged. Please add:' +
+			'\n\n<nowiki>' + params.tagText + '</nowiki>\n\nto the top of the page to complete the nomination. Thank you. ~~~~';
```

Comment:
```suggestion
		var editRequest = '{{subst:xfd edit protected|1=' + pageobj.getPageName() + '|discussion=' + params.discussionpage + '|tag=<nowiki>' + params.tagText + ''\u003C/nowiki>}}'; // U+003C: <
```

I created [the template](https://en.wikipedia.org/wiki/Template:Xfd_edit_protected).

And we can't really use </nowiki> here as that would break the script page on-wiki.

Reply:
Nice; I had created an earlier [draft in my userspace](https://en.wikipedia.org/wiki/User:Amorymeltzer/sandbox/temp), actually lined up pretty closely.  I made a couple of minor tweaks, I like your method of venue determination, much simpler!

> And we can't really use `</nowiki>` here as that would break the script page on-wiki.

Oof, good point!

Reply:
Wow, really close! I thought I was crazy to do things like `~~<noinclude></noinclude>~~` and `<noinclude>|demo=y</noinclude>`. Good to know it's the standard practise.